pkg/operator/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go

```diff
@@ -146,6 +146,8 @@ type klusterletConfig struct {
 	WorkFeatureGates         []string
 
 	HubApiServerHostAlias *operatorapiv1.HubApiServerHostAlias
+
```

Comment:
might worth add a comment to indicate the meaning of this flag.

Reply:
@qiujian16 added the comments of flag meaning.  so I need to start another PR to api repo?

Reply:
PR in api repo https://github.com/open-cluster-management-io/api/pull/297 PTAL

Reply:
+ @haoqing0110 PTAL